mongoengine/document.py

```diff
@@ -233,6 +235,28 @@ def _get_collection(cls):
 
         return cls._collection
 
+    @classmethod
+    def _get_timeseries_collection(cls):
```

Comment:
please add some tests, similarly to capped collection

Reply:
Hi there,
I, added unit test for time-series collection and  wanted to provide a quick update regarding the recent GitHub Actions failure. The issue appears to be related to timing rather than the test implementation itself. While the tests pass successfully in local environment, they occasionally fail in the CI environment due to timing constraints.

Please let me know if there are any adjustments or further steps you’d suggest. I’m happy to help troubleshoot or refine the implementation as needed.

Reply:
I see this error in the CI
`E       pymongo.errors.OperationFailure: The field 'timeseries' is not a valid collection option. Options: { timeseries: { timeField: "timestamp", granularity: "seconds" }, expireAfterSeconds: 5 }, full error: {'ok': 0.0, 'errmsg': 'The field \'timeseries\' is not a valid collection option. Options: { timeseries: { timeField: "timestamp", granularity: "seconds" }, expireAfterSeconds: 5 }', 'code': 72, 'codeName': 'InvalidOptions'}`

My guess is that the timeseries is not compatible with older version of MongoDB so you need to decorate the tests with e.g `requires_mongodb_gte_44`

Reply:
thanks for your feedback, i would use this decorators, but actually mogodb starts to  supports timeseries collection from version 5.0, [MongoDB 5.0 in the official MongoDB documentation](https://www.mongodb.com/resources/products/capabilities/mongodb-time-series-data), So To ensure the codebase remains scalable and future-proof, I think it would be beneficial to introduce a new decorator similar to the existing ones for MongoDB version checks.

what do you think, are you agree for adding new decorators like this ```requires_mongodb_gte_50```,

Please let me know your thoughts on this suggestion.

Reply:
Sure , feel free to add as needed

Reply:
Dear @Bagerard
For now all is  Done and its ready to merge